src/cpp/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@
 # set minimum version
 cmake_minimum_required(VERSION 3.4.3)
 
-project (RSTUDIO_CPP)
+project("RStudio 1.4")
```

Comment:
How about "RStudio ${RSTUDIO_RELEASE_NAME}"? (then we won't have to remember to bump this every release)